src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const midTerm = 3;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const midTermDiscount = 20;
+  const totalCost = pricePerDay * days;
```

Comment:
you should change `const totalCost` to `let totalCost`; 
change const to let - because `days` is a changing variable;